13-loops/exercises/07-multiplication-table-exercises/02-math-tables/main.go

```diff
@@ -105,4 +104,103 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	args := os.Args[1:]
+
+	if len(args) != 2 {
+		fmt.Println("Usage: [op=*/+-] [size]")
+		return
+	}
+
+	if strings.IndexAny(args[0], "*") == 0 {
```

Comment:
since args[0] is being used multiple times so I suggest to you to store its value in constant and use that constant everywhere

Reply:
Fixed